### ISSUE TITLE
Support alternate Julia commands

### ISSUE THREAD

**Issue by colinxs** [julia-actions/julia-runtest#3]
When testing packages that require a virtual display (e.g. a package that depends on GLFW.jl), tests need to be executed with:
```shell
xvfb-run julia --color=yes --check-bounds=yes --project -e 'using Pkg; Pkg.test(coverage=true)'
```
Would it be possible to roll this into `julia-runtest` somehow?

**Comment by davidanthoff** [comment]
My workaround for this exact problem has been the combination of [this](https://github.com/davidanthoff/Electron.jl/blob/master/src/Electron.jl#L7) with [this](https://github.com/davidanthoff/Electron.jl/blob/master/test/runtests.jl#L5). Not super elegant, but it does the job.

I'm wondering whether `xvfb` is the only example where we would need that? If so, an alternative would be to create another action that just starts `xvfb` and sets the `DISPLAY` env var, I think that would also work?

**Comment by SaschaMann** [comment]
How about adding a config option `xvfb-run` to `action.yml`? Or, more general, some `prefix` option that's inserted in front of the Julia command?

**Comment by davidanthoff** [comment]
Yeah, I can see the value of that. I would prefer the `prefix` option in that case.

The reason I didn't pursue this is my package butler story: in that one the butler creates the CI config files and maintains them, so they can't have any package specific parts. Or at least I haven't figured out a good story for that. But I think that shouldn't stop us from adding such a prefix option here.

**Comment by SaschaMann** [comment]
> in that one the butler creates the CI config files and maintains them, so they can't have any package specific parts. Or at least I haven't figured out a good story for that.

Some apps similar to the butler have config files in the repo to overwrite default values if necessary.

**Comment by davidanthoff** [comment]
Yeah, I already have support for a config file. But I'd like to keep it lean :) Probably not really doable...

**Comment by NLaws** [comment]
Would adding support for `Pkg.test` `test_args` fit under this issue?

**Comment by timholy** [comment]
The links in https://github.com/julia-actions/julia-runtest/issues/3#issuecomment-558826475 were not permalinks. Do you mean https://github.com/davidanthoff/Electron.jl/blob/67f7a980005e48a730ebae64c6b449a1d9c01b26/src/Electron.jl#L17-L24 and https://github.com/davidanthoff/Electron.jl/blob/67f7a980005e48a730ebae64c6b449a1d9c01b26/test/runtests.jl#L6?

Would love to have an easier solution for this, but these hints are very helpful.

**Comment by timholy** [comment]
Ah...except it doesn't work for Gtk. But I see @SaschaMann has implemented this in #4.
